specifications/api/swagger.yaml

```diff
@@ -3734,6 +3954,7 @@ x-tagGroups:
     - Proxy
     - Settings
     - Experimental
+    - Shared Wallets
```

Comment:
This is not the best place for this :grimacing: , let's have a new section about "Shared Wallet", they ain't misc!

```yaml
- name: Shared Wallets
  tags: 
  - Shared Wallets
  - Shared Assets
  - Shared Addresses
  - etc...
```

Reply:
done